buildSrc/opensearch-hadoop-version.properties

```diff
@@ -1,4 +1,4 @@
-opensearch_hadoop        = 3.0.0
-opensearch      = 3.0.0
+opensearch_hadoop        = 1.0.0
+opensearch      = 1.3.8
```

Comment:
We don't want an impedance mismatch of OpenSearch and Lucene here. The version of Lucene should match the version that is being used for OpenSearch 1.3.x

Reply:
Lucene | OpenSearch | ES
-- | -- | --
9.4.0 snapshot | main / 3.0.0 | main / 8.5.0
9.3.0 | 2.x / 2.32.2 | 8.4
9.2.0 | 2.1 | 8.3
9.1.0 | 2.0 | 8.2
9.0.0 |   | 8.18.0
8.11.1 |   | 7.17
8.10.1 | 1.x / 1.41.31.2 | 7.16
8.9.0 | 1.1 | 7.157.14
8.8.2 | 1.0 | 7.13
8.8.0 |   | 7.12
8.7.0 |   | 7.117.10
  |   |  
8.6.2 |   | 7.9
8.5.1 |   | 7.87.7
8.4.0 |   | 7.6
8.3.0 |   | 7.5
8.2.0 |   | 7.4
8.1.0 |   | 7.3
8.0.0 |   | 7.27.1
7.7.3 |   | 6.8